cmd/cri-containerd/options/options.go

```diff
@@ -35,6 +35,17 @@ type ContainerdConfig struct {
 	ContainerdSnapshotter string `toml:"snapshotter"`
 	// ContainerdEndpoint is the containerd endpoint path.
 	ContainerdEndpoint string `toml:"endpoint"`
+	// Runtime is the runtime to use in containerd. We may support
+	// other runtimes in the future.
+	Runtime string `toml:"runtime"`
```

Comment:
Follow the pattern? `ContainerdRuntime`?
And the other 3 configs.

Reply:
I don't mind prefixing with Containerd .. so many comments on what to prefix variable names with.. :-)

Reply:
But I'd rather remove the prefix since we are already in the ContainerdConfig struct

Reply:
See https://github.com/kubernetes-incubator/cri-containerd/pull/275#discussion_r141168532.